Add Utility to Send Deposit Transactions

nisdas opened the pull request:
This allows us to send multiple deposit transactions with the deposit data to the contract , so that we can test more easily. Instead of having to use metamask / web3 to achieve this. The command to use this  is 

```
bazel run //contracts/deposit-contract/sendDepositTx -- --httpPath=https://goerli.prylabs.net --keystoreUTCPath /path/to/keystore --passwordFile /path/to/password --depositDelay 2  --depositContract 0x767E9ef9610Abb992099b0994D5e0c164C0813Ab
```

We can also either send all the transactions at once or add in a delay between them, we can specify the size of the deposits and the number of deposits also.

rauljordan commented:
Is this utility just to test out chainstart? because to test out the rest of the system we'd still require to create 8 validator accounts with their deposit data and issue 8 separate tx's with that deposit data, right?

nisdas commented:
Ah no its just to send a deposit transaction, we can also use it to test chainstart as the default value is 8 transactions.this generates fake deposit data where it will generate a validator account and then serialize the pubkey along with the other values. that we send to the contract. 

We also can use it for other forms of testing where we can have it continously send transactions to the contract to simulate a live system, or some other scenario

prestonvanloon commented on contracts/deposit-contract/sendDepositTx/BUILD.bazel:
```diff
@@ -0,0 +1,61 @@
+        "@com_github_x_cray_logrus_prefixed_formatter//:go_default_library",
+        "@io_k8s_apimachinery//pkg/apis/meta/v1:go_default_library",
+        "@io_k8s_client_go//kubernetes:go_default_library",
+        "@io_k8s_client_go//rest:go_default_library",
```
Why does the image target have a lot of deps that aren't present in the binary?

prestonvanloon replied:
This should be the same set of deps. Please ensure this manual target builds too.

```
bazel build //contracts/deposit-contract/sendDepositTx:image
```

Do we even need a docker image for this tool?

nisdas replied:
ah, we do not need a docker image for this tool. It's supposed to be just for testing, and will not be used in production. For some reason `gazelle` didn't remove them

nisdas replied:
Just realised I never removed the image in the build file

prestonvanloon replied:
Ok, please remove this target :)

nisdas replied:
Done !

terenc3t commented:
I think this tool is really great but doesn't test production level deposit behavior where every tx has different latencies. We should be using both methods so no big gets lost : )

nisdas commented:
@terenc3t For variable transaction times , I added in a folded t-distribution which we can use to generate random values to simulate real -world ish transactions. The reason I used a t-distribution is due to the small number of transactions that we will be sending though this utility, so `n` is small. Also the tails are fatter in t distributions ,so we get larger amount of variability for extreme values. Since negative transaction times do not make sense, it is a 'folded' distribution as we take the absolute of the generated values. I have added flags to specify all that is needed to generate the t distribution. Let me know if you think another stat distribution may be better for this.

@prestonvanloon I made the changes, PTAL
